app/client/cli/actor.go

```diff
@@ -0,0 +1,315 @@
+package cli
```

Comment:
An issue I just realized is that validators (and maybe fisherman tentative the question I asked in #311) don't have relay chain IDs. I believe they will just be ignored so things will function E2E, but it'llbe very confusing to the user that they can pass that in.

I was thinking of just updating the docs saying "unneeded params will be ignored", but it still feels misleading and unclear. 

Still thinking if we should create a follow up ticket for it or if there's a quick & easy fix here...

Reply:
@deblasis Can you create a ticket to track this?

Reply:
@Olshansk, @andrewnguyen22 : I was writing the ticket and meanwhile, I thought:

I guess this issue is similar to https://github.com/pokt-network/pocket/pull/177#discussion_r1001113378 with the exception that for that one we already have a dependency that could unlock it which is the keybase.

Since it looks like the end-to-end scenario is what we want to achieve short-term and that I added these arguments in the first place when I was more of a newbie just by referencing V0, shall we just remove them and add them back when the time comes to keep it stupid simple? 

Ergo: 
- when we know which actors require a relay chain id and for which commands in V1, we'll add to the CLI and RPC accordingly 
- similarly, when we'll have a keybase, we'll accept a `fromAddr` argument where needed

Reply:
I'm okay with either approach of the following approach

1. Leaving a comment + TODO + explanation of why the var isn't being used
2. Removing it altogether

Feel free to take either one depending on your best judgment and time/work capacity

Reply:
Resolving this per the TODOs left alongside `cobra.ExactArgs`